NEWS.md

```diff
@@ -0,0 +1,40 @@
+# shiny.router (development version)
+
+- Changed the dots (`...`) argument in `router_ui()` to allow dynamically passed arguments
+- Fixed the issue where the 404 page was not working when a non-valid page is opened as the first page
+
+# shiny.router 0.3.0
+
+- Added a new API that is compatible with Rhino
+- `router_ui` and `router_server` replaced `make_router`
+- `router_ui` should be applied inside the UI function (not outside, like `make_router`) and thanks to that, can utilize `ns()` if used in a Rhino app
+- `router_server` adds all required server components (mostly `observeEvents`) to the server part of the application
+- Marked `make_router` as deprecated, it is not removed
+
+# shiny.router 0.2.3
+
+- Applied `shiny::createWebDependency()` before `renderDependencies()`
```

Comment:
This is not a good description - this informs how the bug was solved, but tells nothing about what the bug was. Again, you need to read the PRs - in this particular case, you would find the link to the issue with bug description. Based on that, create an entry for the changelog.